Rematch/Addon.lua

```diff
@@ -103,13 +164,24 @@ function RematchPlugin:OnDisable()
 
     self:UnhookAll()
 
+    if rematchVersion >= ns.Version:New(5, 0, 0, 0) then
+        Rematch.events:Unregister(self, 'REMATCH_TEAMS_CONVERTED')
+        Rematch.events:Unregister(self, 'REMATCH_TEAM_OVERWRITTEN')
+        Rematch.events:Unregister(self, 'REMATCH_TEAM_UPDATED')
+        Rematch.events:Unregister(self, 'REMATCH_TEAMS_WIPED')
+        Rematch.events:Unregister(self, 'REMATCH_TEAM_DELETED')
```

Comment:
Do you want to unregister this event? Will run into the possibility of having an orphaned script. And since Rematch5 will reuse teamIDs, there's a chance, once enabled again, of a new team being assigned a script. 

I don't see this happening, but if it does happen, it's not really breaking. A user can just delete the script and start over.

Reply:
I think most users are like me and didn't even know you can disable the plugin, or would even want to.